go/vt/sqlparser/ast_format.go

```diff
@@ -1574,20 +1574,20 @@ func (node *CreateTable) Format(buf *TrackedBuffer) {
 
 // Format formats the node.
 func (node *CreateView) Format(buf *TrackedBuffer) {
-	buf.WriteString("create")
+	buf.astPrintf(node, "create %v", node.Comments)
 	if node.IsReplace {
-		buf.WriteString(" or replace")
+		buf.WriteString("or replace ")
```

Comment:
I assume there's no requirement to have a space after a comment in the create statement?

Reply:
That's correct! And also it is more inline with the rest of our `ast_format` code, where comments have no trailing spaces.